src/content/app/species/components/species-title-area/SpeciesTitleArea.module.css

```diff
@@ -41,12 +41,14 @@
 }
 
 .assemblyCountWrapper {
+  cursor: pointer;
```

Comment:
- is the whole wrapper supposed to be clickable, or just the blue "Assemblies"?
- you wouldn't need this line if you turned your span (or your div) into an html button element